tests/Fixer/NamespaceNotation/NoBlankLinesBeforeNamespaceFixerTest.php

```diff
@@ -18,7 +18,7 @@
 use PhpCsFixer\WhitespacesFixerConfig;
 
 /**
- * @author Graham Campbell <graham@alt-three.com>
+ * @author Graham Campbell <hello@gjcampbell.co.uk>
```

Comment:
how this got here?

Reply:
by typing

Reply:
sorry, I don't get it too, @SpacePossum ?

Reply:
not helping reading the diff for sure :shrug:

Reply:
it will, if you see my PHP8.1 fix after rebasing on this and bunch of other PR we can finally discuss meaningful changes and not meta and semantics and cleanups